modules/vba_documents/app/models/vba_documents/upload_submission.rb

```diff
@@ -31,10 +31,12 @@ class UploadSubmission < ApplicationRecord
     VBMS_IMPLEMENTATION_DATE = Date.parse('28-06-2019')
     FINAL_SUCCESS_STATUS_KEY = 'final_success_status'
     IN_FLIGHT_STATUSES = %w[received processing success].freeze
+    IN_PROGRESS_STATUSES = %w[received processing].freeze
```

Comment:
Does `IN_PROGRESS_STATUSES` need to include any of the statuses from `ALL_STATUSES`? Would it be safer to create a `SUCCESS_STATUS = 'success'` and then change the `in_progress` scope to:
 `scope :in_progress, -> { where.not(status: SUCCESS_STATUS) }` 
that way we don't miss any other statuses that might need to be there? Thoughts?